README.md

```diff
@@ -84,14 +96,33 @@ or
 
 ### TypeScript
 
-In order to use this config in TypeScript project make sure you have installed following dependencies:
+Extend your ESLint config using `plugin:import/typescript`, for example:
 
-* [`@typescript-eslint/eslint-plugin`](https://yarnpkg.com/en/package/@typescript-eslint/eslint-plugin)
-* [`@typescript-eslint/parser`](https://yarnpkg.com/en/package/@typescript-eslint/parser)
-* [`typescript`](https://yarnpkg.com/en/package/typescript)
+```json
+{
+  "extends": [
+    "@callstack",
+    "plugin:import/typescript"
+  ]
+}
+
+```
+
+If you are using VSCode, add to `.vscode/settings.json`:
```

Comment:
Probably better to configure globally: `Code > Preferences > Settings`

Reply:
Not all my projects use ESLint, so I am strongly unsure about this. Moreover from the user perspective - linter for pure JS works fine, so why should I add this in this case?

Reply:
> Still, I am against changing global settings

If your project doesn't use ESLint, it just wouldn't do anything. It avoids configuring editor for every project again separately.

Anyway, it's just personal opinion. It should be documented on how to configure globally as well as local and let the user do what they want to do.

Reply:
I agree, we should document how to cmconfigure it locally and globally.

Reply:
Let's add a note stating that "alternatively, this configuration can be done globally, ..."